lib/energy_sparks/csv_loader.rb

```diff
@@ -0,0 +1,25 @@
+require 'csv'
+module EnergySparks
+  class CsvLoader
+    def self.default_options
```

Comment:
Now we have set this up, at some point I guess we have some other CSV parsers which we should switch to using this....

Reply:
Yes, as we come across them I thought we could move them across.